components/ErrorDisplay/__tests__/__snapshots__/ErrorDisplay.test.js.snap

```diff
@@ -2,22 +2,22 @@
 
 exports[`ErrorDisplay should render with just required props 1`] = `
 <div
-  className="ErrorDisplay"
+  className="ErrorDisplay bg-[#121212] w-[100% h-[100%]"
 >
   <div
-    className="bg"
+    className="bg bg-[url(\\"/static/images/TankFlip.gif\\")] md:h[100vh] flex flex-col justify-center w-[100%] text-white bg-cover items-center"
   >
     <div
-      className="textContainer"
+      className="text-center m-[1rem]"
```

Comment:
```suggestion
      className="text-center m-4"
```

Try using the token values. It will make for a faster build-time process when we have fewer custom properties (values that are defined in square brackets).